Hazel/src/Hazel/Core/KeyCodes.h

```diff
@@ -1,5 +1,142 @@
 #pragma once
 
+namespace Hazel
+{
+    enum class Keys
```

Comment:
When using this enum, we write `Keys::A`. Maybe create a typedef to map Keys on Key so we can write `Key::A`?
```cpp
typedef enum class Keys
{
    ... // key codes
} Key;
```
or
```cpp
enum class Keys
{
    ... // key codes
};
using Key = Keys;
```